core/templates/components/profile-link-directives/profile-link-image.component.ts

```diff
@@ -59,9 +55,9 @@ export class ProfileLinkImageComponent implements OnInit {
     // image if user is not logged in or has not uploaded a profile
     // picture, or the player is in preview mode.
     this.profileLinkImageBackendApiService.fetchProfilePictureData(
-      this.profileImageUrl).then((response: ProfileDict) => {
+      this.profileImageUrl).then((profilePicture) => {
```

Comment:
profilePictureString perhaps, or something to indicate that this is not some kind of object.

Also -- no typeinfo?

Reply:
Done!